frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { ObjectDataReductionBody } from './object-data-reduction-card-body';
+import { ObjectDataReductionItem } from './object-data-reduction-card-item';
+import './object-data-reduction-card.scss';
+
+const ObjectDataReductionQueries = {
+  EFFICIENCY_QUERY: 'NooBaa_reduction_ratio',
+  SAVINGS_QUERY: 'NooBaa_savings',
+};
+
+const DataReductionCard: React.FC<DashboardItemProps> = ({
+  watchPrometheus,
+  stopWatchPrometheusQuery,
+  prometheusResults,
+}) => {
+  React.useEffect(() => {
+    Object.keys(ObjectDataReductionQueries).forEach((key) =>
+      watchPrometheus(ObjectDataReductionQueries[key]),
+    );
+    return () =>
+      Object.keys(ObjectDataReductionQueries).forEach((key) =>
+        stopWatchPrometheusQuery(ObjectDataReductionQueries[key]),
+      );
+  }, [watchPrometheus, stopWatchPrometheusQuery]);
+
+  const efficiencyQueryResult = prometheusResults.getIn([
+    ObjectDataReductionQueries.EFFICIENCY_QUERY,
+    'result',
+  ]);
+  const savingsQueryResult = prometheusResults.getIn([
+    ObjectDataReductionQueries.SAVINGS_QUERY,
+    'result',
+  ]);
+
+  const efficiency = _.get(efficiencyQueryResult, 'data.result[0].value[1]', null);
+  const savings = _.get(savingsQueryResult, 'data.result[0].value[1]', null);
+  const logicalSize = _.get(savingsQueryResult, 'data.result[0].metric.logical_size', null);
+
+  const efficiencyProps = {
+    title: 'Efficiency Ratio',
+    value: efficiency,
+    infoText: 'Efficiency ratio refers to the deduplication and compression process effectiveness.',
+  };
+
+  const savingsProps = {
+    title: 'Savings',
+    value: savings,
+    logicalSize,
+    infoText:
+      'Savings shows the uncompressed and non-deduped data that would have been stored without those techniques.',
+  };
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Object Data Reduction</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody>
```

Comment:
you dont even need `ObjectDataReductionBody`
```suggestion
      <DashboardCardBody isLoading={!savingsQueryResult && !efficiencyQueryResult}>
```